platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -369,13 +369,18 @@ function _getViewportComponent(displaySets, viewportComponents) {
   const SOPClassHandlerId = displaySets[0].SOPClassHandlerId;
 
   for (let i = 0; i < viewportComponents.length; i++) {
+    if (!viewportComponents[i]) throw new Error("viewport components not defined");
```

Comment:
This set of changes makes it a lot easier to debug what is going on for new modes - they often don't quite get loaded correctly.

Reply:
To be consistent with the rest of the codebase, can we have brackets for the if statements?